crates/katana/core/src/backend/mod.rs

```diff
@@ -488,3 +494,13 @@ fn pretty_print_resources(resources: &ResourcesMapping) -> String {
 
     mapped_strings.join(" | ")
 }
+
+fn trace_events(events: &[Event]) {
+    for e in events {
+        let mut formattedEventData = String::new();
+        for (keyNumber, k) in e.content.keys.iter().enumerate() {
+            formattedEventData.push_str(&format!("key{}={} ", keyNumber + 1, k.0));
+        }
+        trace!("Event emitted {}", formattedEventData);
```

Comment:
prefer the log to be like this, so that the log doesnt get too long if there are many keys

```console
[2023-08-02T17:06:22Z TRACE katana_core::backend] Event emitted keys=[0x006b1c9603e4bb44f33f1ab79ad2fc0a3d631204beebcd32a1f31b00eb999f80, 0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0, 0x03ae9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0]
```